.travis.yml

```diff
@@ -448,8 +456,31 @@ before_script:
       *)           echo "Unsupported platform '$PLATFORM'"; exit 1 ;;
     esac
 
+  - cd ..
+  - mkdir cmocka
+  - cd cmocka
+  - PTH=$(pwd)
+  - cd ..
+  - export PATH="$PTH:$PATH"
+  - export LD_LIBRARY_PATH="$PTH/lib"
+  - CFLAGS="-W -Wall -O2 -g -I$PTH/include/"
```

Comment:
This breaks prepared CFLAGS from above PLATFORM switch.

Reply:
Fixed.